### DISCUSSION TITLE
11.x demo URL admin access throwing 500 error

### DISCUSSION THREAD

**Discussion by opbissa** [pimcore/pimcore#14154] [General]
While trying to access Pimcore 11 demo on https://11.x-dev.pimcore.fun/admin URL I am getting "500 internal server error" after entering the login credentials.

**Comment by mattamon** [comment]
Hey  @opbissa,

we are aware of this issue. There is an open PR that needs to be merged into the CMF.
https://github.com/pimcore/customer-data-framework/pull/404
I am going to notify you, when it works again :)

**Reply by mattamon** [reply]
Hey @opbissa,

the Demo now works again. :)
